packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -529,7 +529,9 @@ describe('KeyringController', () => {
       await withController(async ({ controller }) => {
         expect(controller.isUnlocked()).toBe(true);
         expect(controller.state.isUnlocked).toBe(true);
-        controller.setLocked();
+
+        await controller.setLocked();
+
```

Comment:
This is a visible effect of the changes: 

```typescript
expect(controller.isUnlocked()).toBe(false);
```
The missing `await` breaks the assertion